app/models/mixins/ems_refresh_mixin.rb

```diff
@@ -0,0 +1,59 @@
+module EmsRefreshMixin
+  extend ActiveSupport::Concern
+
+  included do
+    supports :ems_refresh
+  end
+
+  class_methods do
+    def queue_refresh(ems_id, ems_ref)
```

Comment:
TODO handle `Class.queue_refresh([id1, id2, ...])` which is used by Host and VmOrTemplate

Reply:
Do we actually use these APIs?  I know I wrote some of them for developer convenience.

Reply:
Yeah I'm tracking down where these are called from, I've seen `object.refresh_ems` used not sure about `Class.refresh_ems([ids])`